tests/conftest.py

```diff
@@ -1131,11 +1131,14 @@ def cluster_teardown_finalizer():
         log.info("Will teardown cluster because --teardown was provided")
 
     # Download client
-    force_download = (
-        config.RUN["cli_params"].get("deploy")
-        and config.DEPLOYMENT["force_download_client"]
-    )
-    get_openshift_client(force_download=force_download)
+    if config.DEPLOYMENT["skip_download_client"]:
+        log.warning("Skipping client download")
```

Comment:
```suggestion
        log.info("Skipping client download")
```
If it's a legitimate config item, it doesn't warrant a warning IMO

Reply:
Done!